Updated Lifecycle.md to improve flow

victoronline opened the pull request:
This is a first pass. Updated to improve flow.

victoronline committed:
Merge pull request #1 from planetarysoftware/master

syncing fork

victoronline committed:
Merge pull request #2 from planetarysoftware/master

resync fork

victoronline committed:
Updated Lifecycle.md to improve flow

Updated to improve flow of application and approval process. This is the first pass.

victoronline committed:
Updated Lifecycle.md to denote issue created at review step

Forgot to denote issue created at review step to create track of comments.

victoronline committed:
Merge pull request #3 from victoronline/victoronline-patch-4

Updated Lifecycle.md to denote issue created at review step

rbeyer commented on Lifecycle.md:
```diff
@@ -34,12 +34,12 @@ chartered by the TC of the project they exist under.
+	* If the application is accepted by the TSC, the TL* enters Incubation
+	* A mentor(s) is assigned tor work the TC to get the TL* to ensure requirements are met
+	* Once requirements are met, the TSC votes to graduate the TL* from Incubation
+	* The TL* is now a fully authorized Planetary Software TL*
```
I know these were like this in the parent document (but I think that was an oversight), these probably don't need to be verbatim text, and probably can be 'regular'.

victoronline replied:
OK. Do we care about numbering these steps or just having them in order? I originally removed the numbers to not confuse with the numbered steps I added below.

rbeyer replied:
I don't think so.  If you think they look better as bullets, go for that.

victoronline replied:
Ok. Changed but kept bullets.

jessemapel reviewed (commented):
I agree with Ross about the formatting. Content looks great :+1:

jlaura commented on Lifecycle.md:
```diff
@@ -34,12 +34,12 @@ chartered by the TC of the project they exist under.
+	* Like-minded individuals form a TC in order to establish a TL*
+	* That TC establishes a charter for their TL* and submits an application to the Planetary Software TSC
+	* If the application is accepted by the TSC, the TL* enters Incubation
+	* A mentor(s) is assigned tor work the TC to get the TL* to ensure requirements are met
```
/s/tor/to/

jlaura commented on Lifecycle.md:
```diff
@@ -68,31 +68,33 @@ For more information (and examples) of how to write a charter for a TL*,
+The process to join the Planetary Software Organization
 as a TLP or TLWG is as follows.
 
+### 1. Application Completed - Pull Request Submitted
```
I am finding the 'Application Completed' to be weird here. If the PR is submitted is the application complete? It seems like that just starts the process.?

rbeyer replied:
Yeah, maybe "Submit Application" here?

victoronline replied:
Updated. Do we like the wording?

jlaura commented on Lifecycle.md:
```diff
@@ -68,31 +68,33 @@ For more information (and examples) of how to write a charter for a TL*,
 Copy the [Application Template](Applications/Template.md), fill
 it out, add the file to the [Applications](Applications/)
-directory, and submit that as a Pull Request.
+directory and submit this as a Pull Request.
```
I'm not a fan of this/that here. What about 'and submit the filled in Template.md as a Pull Request to this repository'?

victoronline replied:
Added suggested text

jlaura commented on Lifecycle.md:
```diff
@@ -68,31 +68,33 @@ For more information (and examples) of how to write a charter for a TL*,
 
-The Application is considered 'accepted' when the Pull Request is
-merged by the TSC.
+### 2. Pull Request is Merged - Application is Accepted
```
Maybe the above 'Application Complete' could be 'Application Submission' or something? Then #2 makes more sense I think.

victoronline replied:
Changed text to 'Submit Application'

jlaura commented on Lifecycle.md:
```diff
@@ -262,7 +260,7 @@ meeting).
 The TLP or TLWG is considered to be chartered once that PR lands.
 
-### Badges
+#### Badges
```
badges - the reason we are all in this thing.

victoronline committed:
Updated Application Complete - Submitted text

Updated Application Completed text to Submit Application for clarity.

victoronline committed:
Update Lifecycle.md

victoronline committed:
Changed verbatim text but kept bullet points 

Below Lifecycle Overview, changed verbatim text to regular text but kept bullet points

victoronline committed:
Added step to create an issue when project enters incubation

Added text to denote when the Issue is created when project enters the incubation step.
